client/payment-methods/index.js

```diff
@@ -328,6 +328,24 @@ const PaymentMethods = () => {
 									onCheckClick={ () => {
 										handleCheckClick( id );
 									} }
+									isPoEnabled={
+										wcpaySettings.hasOwnProperty(
+											'progressiveOnboarding'
+										)
+											? wcpaySettings
+													.progressiveOnboarding
+													.isEnabled ?? false
+											: false
+									}
```

Comment:
You can get advantage of the optional chaining operator (`?.`) to write a shorter version of those. The following code will safely access the nested property falling back to `undefined`, which equals to a _falsy_ value in a boolean comparison.

```js
wcpaySettings?.progressiveOnboarding?.isEnabled
```

Reply:
Thanks! For some reason I thought that was a TS thing (and this is a `js` file) - TIL I can do this in regular JS too!

Reply:
Yes, it can be easy to miss what's from TS and JS 😄  In this case, the optional chaining operator is from [ES6 specification](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-OptionalExpression).